front-end/src/components/CenterConsole/CenterConsole.js

```diff
@@ -6,21 +6,10 @@ import { Container, Grid } from '@material-ui/core';
 const CenterConsole = (props) => {
   return (
     <Container maxWidth='sm'>
```

Comment:
Just checking again, would it still be okay for me to remove the Container around RecipeList?

Reply:
hey @chaomonica, That would be fine. It does slightly change the mainPage center console by making the cards only slightly larger(not an issue at all). Sorry I did not include it, but no problem for you to rely on it being removed. I'll leave it removed on my end just to make sure there are no issues going forward